master/internal/experiment.go

```diff
@@ -945,6 +946,10 @@ func (e *experiment) setRP(ctx *actor.Context, msg sproto.SetResourcePool) error
 func (e *experiment) toV1Job() (*jobv1.Job, error) {
 	workspace, err := workspace.WorkspaceByProjectID(context.TODO(), e.ProjectID)
 	if err != nil {
+		// FIXME: DET-9563 workspace and/or project is deleted.
+		if err == sql.ErrNoRows {
+			return nil, nil
```

Comment:
I don't love this ```return nil, nil``` since it is easy to miss this is a possibility from the function definition 

I think it is fine though

Reply:
hmm if this isn't common practice I can move this part up to where we're calling it